### PR TITLE
Website/ Slider baseline performance + improve front-page SSR

### PR DESCRIPTION
**Pull request by DavidBachmann**
[x] Hero slider is way too heavy on initial loading times
[x] Hero causes page jumping
[x] Initial render (SSR) of layout is broken

### Hero animations are being _temporarily_ removed to establish a baseline performance and to improve the homepage ASAP. Animations will be put back in using a different strategy soon.

No cache on production build comparisons on fast 3G simulation:

**Before**
before 2020-10-28 09_02_41

**After**
after 2020-10-28 09_02_16

We shave off 2 seconds and all the jumping but as you can see there's a lot of other improvements to be made.

Edit: Improved server-side rendered layout
**Before:**
Screenshot 2020-10-28 at 10 06 11

**After**
Screenshot 2020-10-28 at 10 05 50

### CODE REVIEW

File: apps/web/components/FrontpageTabs/illustrations/Illustration.tsx [hunk 1]
```diff
@@ -0,0 +1,420 @@
...
+  illustrationIndex?: number
+}
+
+const illustrations = [
+  dynamic(() => import('./ManWithPhoneIllustration')),
+  dynamic(() => import('./WomanWithLaptopIllustration')),
+  dynamic(() => import('./WomanFeedingBabyIllustration')),
+  dynamic(() => import('./ManWithStrollerIllustration')),
+  dynamic(() => import('./WomanOnBusIllustration')),
+  dynamic(() => import('./ManOnBenchIllustration')),
```
Comment (arnifreyrs) [inline apps/web/components/FrontpageTabs/illustrations/Illustration.tsx:16]: Are you going to be extracting these illustrations into another shared lib in a future PR?

**Review by stefanhar — DISMISSED** [review]
LGTM

**Review by arnifreyrs — APPROVED** [review]

File: apps/web/components/FrontpageTabs/illustrations/Illustration.tsx (hunk 1 above)
Comment (DavidBachmann) [reply apps/web/components/FrontpageTabs/illustrations/Illustration.tsx:16]: These illustrations won't be used in FrontpageTabs in the future. For other use cases we'll move them into island-ui/assets or similar, later.

**Review by jeremybarbet — APPROVED** [review]
LGTM

**Comment by DavidBachmann** [comment]
Build froze so I used the opportunity to push my SSR-fixes to this PR.

**Review by jeremybarbet — APPROVED** [review]

**Review by arnifreyrs — APPROVED** [review]

**Review by stefanhar — APPROVED** [review]
LGTMA

**Pull request merged**